hydra-packet-importer/import.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+import json
+import packet
+import base64
+from pprint import pprint
+import subprocess
+import sys
+
+def debug(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+
+def get_devices(manager):
+    devices = []
+
+    page = 'projects/%s/devices?page=%d' % (config['project_id'], 1)
```

Comment:
A bit more idiomatic python3:

```suggestion
    page = 'projects/{}/devices?page={}'.format(config['project_id'], 1)
```

Reply:
You bet!